### ISSUE TITLE
More input types

### ISSUE THREAD

**Issue by driesvints** [blade-ui-kit/blade-ui-kit#13]
Provide more components for different input types. Not saying we should provide a component for every single one, just the ones that make sense. Like: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color

**Comment by abenettt** [comment]
I think it it would be good to use the default color type for the component, this would not only be semantically correct it would also remove another dependency.

**Comment by driesvints** [comment]
@abenettt yeah maybe. But Pickr allows for much more than browser defaults. So not sure if we should remove that.

**Comment by abenettt** [comment]
I feel like that in most cases the default color picker is sufficient, maybe we can implement the native color picker as the default and use `Pickr` as an optional opt in.
